components/widget/Message.tsx

```
@@ -112,15 +112,57 @@ export default function Message({
             key={attachment.url}
-            href={attachment.url}
-            className="block p-4 pt-0"
-            target="_blank"
-            rel="noopener noreferrer"
+            className="p-4 pt-0 cursor-pointer"
+            onClick={() => {
+              // Open image in new tab with proper handling
+              const newWindow = window.open();
+              if (newWindow) {
+                // Properly escape values to prevent XSS
+                const safeTitle = document.createElement('div');
```

ESLint

[error] Replace `'div'` with `"div"`

(prettier/prettier)